aspnet/Gantt/How-to/Change-Weekend.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Change-Weekend | Gantt | ASP.NET Webforms | Syncfusion
-description: Change workweek/working days in a week
+title: Change Weekend | Gantt | ASP.NET Webforms | Syncfusion
+description: Learn here about how to change workweek/working days in Syncfusion Essential ASP.NET Gantt Control, its elements, and more.
 platform: aspnet
 control: Gantt
 documentation: ug
 ---
 
-## Change workweek/working days in a week
+# Change workweek/Weekend in ASP.NET Gantt 
```

Comment:
Change like this - Change workweek/Weekend in ASP.NET WebForms Gantt

Reply:
Done